nginx/nginx.conf

```diff
@@ -0,0 +1,40 @@
+# This is based on the default nginx.conf file in the base image (at
+# /etc/nginx/nginx.conf)
+
+worker_processes auto;
+error_log /dev/stderr info;
```

Comment:
Taken from `nginx/conf/http/logging.conf`, but set at the root (`main`) context instead of `http` (previously the root context was still using a log file). Captures more events (particularly at start-up).